site/book/04-using-functions/02-imperative-function-execution.md

```diff
@@ -158,7 +158,7 @@ Here is an example:
 $ kpt fn source wordpress \
   | kpt fn eval - --image gcr.io/kpt-fn/set-namespace:v0.1 -- namespace=mywordpress \
   | kpt fn eval - --image gcr.io/kpt-fn/set-labels:v0.1 -- app=wordpress env=prod \
-  | kpt fn sink wordpress
+  | kpt fn sink hydrated-wordpress
```

Comment:
We don't used `hydrated` term in user-facing docs. May be call it `my-wordpress` or something.

Reply:
Done.